userguide/subscription/includes/catalog-core.adoc

```diff
@@ -355,20 +355,26 @@ In the remainder of this section we illustrate each type of rule supported by th
 
 === Billing Alignment Rules
 
-This rule uses the Creation Context and configures the type of billing alignment used with a subscription. There are three kinds of alignment available:
+Billing alignment is concerned with specifying the day on which a particular subscription is to be billed. If an account has multiple subscriptions it is often desirable that they be billed on the same day.
 
-* ACCOUNT - this alignment means that the billing cycle of the subscription will be lined up with the bill cycle day of the account (if you didn't specify a value, the system will generate one using the first recurring bill date of all subscriptions with an `ACCOUNT` billing alignment). In some cases, this is undesirable because it means that a proration charge will be applied on first billing to line up the cycles.
-* SUBSCRIPTION - this alignment will cause the subscriptions bill cycle to line up with the first bill day of the subscription plan. So, if the subscription starts on January 3rd and has a 15 day free trial, the first billed day with be January 18th and the bill cycle day for the subscription will be set to 18.
-* BUNDLE - this alignment is useful for add-ons because it sets the bill cycle day to be aligned with whatever the base plan is using.
+If the billing period for a subscription is MONTHLY or a multiple of MONTHLY (such as QUARTERLY, ANNUAL, etc.) then we define the Bill Cycle Day (BCD). This is the date on which the billing occurs for that subscription each month. If multiple subscriptions have the same BCD, they may be billed on the same invoice. However, if the BCD for a new subscription differs from its start date, then the initial bill will have to be prorated. This will also occur anytime the BCD for a subscription is changed.
 
-Example:
+If a BCD falls on a date past the end of a given month, such as April 31, it will be set to the last day of that month.
```

Comment:
It might be worth expanding a little bit here, as it is a very common question:
* If the BCD is the 31st, we would use the last day of the month: 01/31, 02/28, 03/31, etc.
* If the BCD is on the 30th: 01/30, 02/28, 03/30, etc.